user/pull-requests.md

```diff
@@ -20,8 +20,10 @@ it can be merged safely because the build was successful.
 Travis CI builds a pull request when it's first opened and when commits are
 added to the pull request throughout its lifetime.
 
-Rather than test the commits from the branches the pull request is sent from, we
-test the merge between the origin and the upstream branch.
+Rather than test the commits that have been pushed to the branch the pull request
+is from, we test the merge between the origin and the upstream branch. To only
+build on push events, you can disable **Build on Pull Requests** from your
```

Comment:
I'd love to say switch off but I noticed in the blog post about this (https://blog.travis-ci.com/2014-03-05-repository-settings/) says disable though. Thats the only consistency i can find.